sds/spine-route-lookup/sds-api.json

```diff
@@ -0,0 +1,219 @@
+{
+  "openapi" : "3.0.0",
+  "servers" : [ {
+    "description" : "Spine Directory Service API",
+    "url" : "https://virtserver.swaggerhub.com/kainos_nhsnia/SDS_API/1.0.0"
+  } ],
+  "info" : {
+    "description" : "Spine Directory Service API",
+    "version" : "1.0.0",
+    "title" : "Spine Directory Service API",
+    "contact" : {
+      "email" : "tharack.ahmed1@nhs.net"
```

Comment:
Don't use your email here - possibly just remove this - or find something more generic - NHS Digital maybe